django/core/mail/backends/base.py

```diff
@@ -15,8 +15,9 @@ class BaseEmailBackend:
            pass
     """
 
-    def __init__(self, fail_silently=False, **kwargs):
+    def __init__(self, fail_silently=False, use_localtime=False, **kwargs):
         self.fail_silently = fail_silently
+        self.use_localtime = use_localtime
```

Comment:
I'm not thrilled about adding `use_localtime` to the BaseEmailBackend. It doesn't apply to a lot of third-party email backends.

I get that you're trying to share code between Django's built-in EmailBackends. I don't have a great solution, other than duplicating the init code to the backends that use it.

Reply:
Looking at the code again, I probably have to move `use_localtime` to `OPTIONS` as you suggested previously. Since the backend is responsible to generate the dates in the emails, this otherwise can not work properly.

Reply:
Just had another look. `use_localtime` is only used in the console backend. I therefore moved that parameter from the `BaseEmailBackend` to the `console.EmailBackend`. I believe that's the most unintrusive solution.

Reply:
I think `use_localtime` would be needed for any backend that calls EmailMessage.message(), which includes Django's smtp, filebased, console and locmem. (And I guess any third-party backends that call it.)

Agreed that `use_localtime` doesn't really feel like a backend connection option.

An alternative would be to leave `EMAIL_USE_LOCALTIME` as a global setting, outside `EMAIL_PROVIDERS` (and continue using that global setting in EmailMessage.message()). That's probably the best for compatibility, but loses the ability to have different `use_localtime` settings for different providers.